services/clojars/clojars.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const { BaseJsonService } = require('../base');
+const { NotFound } = require('../errors');
+const { version: versionColor} = require('../../lib/color-formatters');
+
+module.exports = class Clojars extends BaseJsonService {
+  async handle({clojar}) {
+    const apiUrl = 'https://clojars.org/' + clojar + '/latest-version.json';
+    const json = await this._requestJson(apiUrl);
+
+    if (Object.keys(json).length === 0) {
+      /* Note the 'not found' response from clojars is:
+          status code = 200, body = {} */
+      throw new NotFound();
+    }
+
+    return {
+      message: "[" + clojar + " \"" + json.version + "\"]",
+      color: versionColor(json.version)
+    };
+  }
+
+  // Metadata
+  static get defaultBadgeData() {
+    return { label: 'clojars' };
+  }
+
+  static get category() {
+    return 'version';
+  }
+
+  static get url() {
+    return {
+      base: 'clojars/v',
+      format: '(.*)',
```

Comment:
The old implementation used `(.+)`, I would be inclined to keep it that way. It won't make that much of a difference, but we will avoid capturing an empty string and making a request that we know is incorrect to the Clojars service.

Reply:
:+1: Good spot. I've also changed it in the other services I have touched in this PR as there is no case for empty string there either. I'll try to remember this as I refactor more services.